ex1/ex1.js

```diff
@@ -0,0 +1,97 @@
+function checkUsername(name){
+    if(name === ''){
+        document.getElementById('username_error').innerHTML = "Please enter your username";
+        return false;
+    }
+    else{
+        document.getElementById('username_error').innerHTML = "";
+        return true;
+    }
```

Comment:
```suggestion
        document.getElementById('username_error').innerHTML = "";
        return true;
```
Có lỗi mới vào điều kiện để set lỗi, còn không có thì set về rỗng + return thôi không cần thiết phải else
Tương tự những function khác rút gọn những if else không cần thiết